src/types/ConditionalOrdersUtilsLib.sol

```diff
@@ -30,4 +32,15 @@ library ConditionalOrdersUtilsLib {
         }
         return oraclePrice;
     }
+
+    /**
+     * @dev Reverts call execution with a custom error that indicates to the
+     * watchtower to poll for new order when the next block is mined.
+     */
+    function revertPollAtNextBlock(string memory message) internal view {
+        revert IWatchtowerCustomErrors.PollTryAtBlock(
+            block.number + 1,
+            message
+        );
+    }
```

Comment:
I'd generally discourage the use of revert utilities, such as this, to reduce the layers of indirection required when reviewing the logic within the smart contract to know where I can expect a revert (sure, it's in the name, but it's another function that I then need to drill into to triple check that it indeed does revert). Depending on the optimiser as well and how this gets compiled, this may also result in additional gas consumption.